neqo-transport/src/tracking.rs

```diff
@@ -674,9 +674,8 @@ impl Default for AckTracker {
 
 #[cfg(test)]
 mod tests {
-    use std::collections::HashSet;
-
     use neqo_common::Encoder;
+    use rustc_hash::FxHashSet;
```

Comment:
A comment general to this changeset:

```suggestion
    use rustc_hash::FxHashSet as HashSet;
```

...would make this a smaller diff.

Reply:
Oh neat!